nostr-client-plus/src/bin/publisher.rs

```diff
@@ -144,3 +154,42 @@ async fn run() -> Result<()> {
     println!("Done: {} jobs sent", jobs_sent);
     Ok(())
 }
+
+// Return a new event based on entry, when entry is None it's a
+// PoW event, otherwise it's Classification event
+fn construct_event(entry: Option<RawDataEntry>, sender: Sender) -> UnsignedEvent {
+    let timestamp_now = chrono::Utc::now().timestamp() as u64;
+    let mut job_type = JobType::PoW.job_type();
+    let mut job_type_str = JobType::PoW.as_ref();
+    let mut kv_key = "pow".to_string();
+    let mut raw_data_id = CryptoHash::new(random());
+    // In case entry is not None, it's Classification job, let's
+    // set all related fields with classification entry
+    if !entry.is_none() {
+        let entry = entry.unwrap();
+        job_type = JobType::Classification.job_type();
+        job_type_str = JobType::Classification.as_ref();
+        kv_key = entry.r2_key;
+        raw_data_id = entry._id;
+    }
+    println!("Constructing entry of id: {:?}", raw_data_id);
```

Comment:
This is printing the randomly generated id, but if the event is classification, this is the wrong id